spec/global_id_utils_spec.rb

```diff
@@ -54,7 +54,7 @@
       strs = ['gid://fish/NeonTetra/1', 'gid://fish/SiameseFighting/1']
       gids = strs.map { |str| GlobalID.new(str) }
 
-      expect(GlobalID).to receive(:find_many).with(gids, ignore_missing: true)
+      expect(GlobalID).to receive(:find_many).with(gids, {ignore_missing: true})
```

Comment:
I assume these are the Ruby 3 updates! 👍🏻

Reply:
Yep!